include/ietf_constants.h

```diff
@@ -1521,6 +1521,7 @@ typedef enum {
 	v2N_CHILD_SA_NOT_FOUND = 44, /* RFC 7296 */
 	v2N_INVALID_GROUP_ID = 45, /* draft-yeung-g-ikev2 */
 	v2N_AUTHORIZATION_FAILED = 46, /* draft-yeung-g-ikev2 */
+	v2N_REDIRECT_FAILED = 47, /* I don't know if this the right place for this? */
```

Comment:
No need for a new notify (these are defined by an RFC and are expected to go across the wire).  Just add a global variable to pluto_stats.[hc], similar to pstats_pamauth_started.

Reply:
ok, thank you - I thought I might be threading on sacred IETF ground here :)